compatibility/bazel_tools/testing.bzl

```diff
@@ -718,19 +704,9 @@ excluded_test_tool_tests = [
     # Changes to explicit disclosure Ledger API interface
     # where DisclosedContract.create_arguments(_blob), DisclosedContract.metadata,
     # InclusiveFilters.template_ids are deprecated and replaced by
-    # DisclosedContract.create_event_payload and InclusiveFilters.template_filters respectively
-    {
-        "start": "2.8.0-snapshot.20231025.1",
```

Comment:
I am surprised you can delete that unpunished. It means that the most recent LAPITT will attempt to run against older ledgers and will not have a problem

Reply:
you're right, deleted too much!

Reply:
7b01437ebe45fb2616a2111b80abcf91012972bc